doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +95,39 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
+ 
+In the multi-cloud ecosystem comprised of different security postures and policies, network domains, products, and business partnerships, the responsibility for managing these different cloud environments necessary to support 5G use cases falls to different enterprises, creating new levels of complexities and a new range of security risks. In such an environment, there are additional security principles to be considered.  These principles, see Table 8-1a below, are drawn from the collaboration with the GSMA Fraud and Security Group (FASG).
+  
+| Multi-cloud Security Principle | Description |
+| ------------------------------ | ----------- |
+| Policy synchronization | Consistency in applying the right security policies across environments, services, interfaces, and configured resources |
+| Visibility | A common data model approach to share events and behaviours across all the key compute, storage, network, and applications resources, environments, virtualised platforms, containers and interfaces |
+| Monitoring | Centralisation, correlation, and visualisation of security information across the different cloud environments to provide an end-to-end view and enable timely response to attacks |
+| Automation | Automation of critical activities including cloud security posture management, continuous security assessments, compliance monitoring, detection of misconfigurations and identification and remediation of risks |
+| Access Management | Organisation of a wide range of users including administrators, testers, DevOps, and developers and customers should be organised into security groups with privileges appropriate to different resources and environments |
+| Security Operations Model | Augmentation of security services provided by cloud service providers with the vetted third-party and/or open-source tools and services, all incorporated into the established overall security operations model |
+
+<p align="center"><b>Table 8-1a. Multi-Cloud Principles</b></p>
+
+For telco operators to run their network functions in a multi-cloud environment, and specifically, in public clouds, the industry will need a set of new standards and new security tools to manage and regulate the interactions between multi-cloud participating parties. For instance, it is a good practice to eliminate access to restricted areas rather than merely prohibit it. For instance, the ETSI specification [TS 103 457](https://www.etsi.org/deliver/etsi_ts/103400_103499/103457/01.01.01_60/ts_103457v010101p.pdf) “Interface to offload sensitive functions to a trusted domain” provides extra security requirements for public clouds so as to enable telco operators the option of running network functions in public clouds.
```

Comment:
@walterkozlowski The sense of the sentence "For instance, it is a good practice to eliminate access to restricted areas rather than merely prohibit it" is unclear for me. I think something is missing to understand what it stands for.

Reply:
@karinesevilla: I have changed to "For instance, it is a good practice to eliminate physical access to restricted areas rather than merely prohibit it".  Does it clarify it?

Reply:
@walterkozlowski  Not really, because I don't see how 'eliminate the physical access' can be an example of  'new standards and new security tools'

Reply:
@karinesevilla, when standards like ETSI specification [TS 103 457] are applied defining the clear interface between more and less trusted domains, it will introduce elimination of access.  However, I am happy to remove this sentence (which was proposed by someone else) if it creates confusion.  It was just meant to be an example but it seems not to play its role properly.  I will remove this sentence and reshape a bit the next one (grammatically only).

Reply:
@walterkozlowski It is good for me, thanks